src/frontend/context/AuthContext.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { createContext, useEffect, type PropsWithChildren } from 'react';
+
+import { HttpProvider } from '@navikt/familie-http';
+import type { ISaksbehandler } from '@navikt/familie-typer';
+
+interface IProps extends PropsWithChildren {
+    autentisertSaksbehandler: ISaksbehandler | undefined;
+}
+
+interface AuthProviderExports {
```

Comment:
Det er kanskje mer passende å kalle det "value" da det blir sendt inn som value til provideren.

```suggestion
interface AuthProviderValue {
```